### PR TITLE
task 6

### PR DESCRIPTION
**Pull request by BobbyWhaler**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,19 +1,33 @@
-main {
+.app-body {
   display: grid;
-  grid-template-columns: 600px auto auto;
-  grid-template-rows: 150px 100px auto 300px;
+  grid-template-columns: 200px 700px auto;
```
Comment (P-Copley) [inline src/App.css]: Having your columns as fixed widths this large puts this layout off the side of most screens. You should start mobile first as scale it up as this requires at least a medium sized laptop to not overflow.

File: src/App.css [hunk 2]
```diff
@@ -1,19 +1,33 @@
...
   border-color: black;
+  background-color:aliceblue;
+  height: 500px;
+  overflow: scroll;
+  padding: 2rem;
+  margin: 1rem;
+}
+.comments-container {
+  grid-column: 3 / 4;
+  grid-row: 2 / 4;
```
Comment (P-Copley) [inline src/App.css]: The comments section is really small. This should be given more screen space to read them properly. Make it take the rest of the page at least.

File: src/components/article-feed.jsx [hunk 3]
```diff
@@ -25,7 +25,7 @@ function ArticlesFeed() {
         return (
         <div className="article-feed">
             <div>
-                <ArticleContainer article={currentArticle}/>
+                <ArticleContainer article={currentArticle} setCurrentArticleComments={setCurrentArticleComments}/>
```
Comment (P-Copley) [inline src/components/article-feed.jsx]: You don't need the wrapping div here

File: src/components/article-feed.jsx [hunk 4]
```diff
@@ -38,7 +38,7 @@ function ArticlesFeed() {
         return (
         <div>
         <div>
```
Comment (P-Copley) [inline src/components/article-feed.jsx]: Same as above, this `div` can be removed

File: src/components/comments-container.jsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+    const handleBackClick = () => {
+        setCurrentArticleComments([])
+    }
+
+  return (
+    <div class="comments-container">
+      <h3>Comments</h3>
+      <button onClick={(event) => handleBackClick()}>Close Comments</button>
+      <div>
```
Comment (P-Copley) [inline src/components/comments-container.jsx]: Use a `ul` or `ol` if you're rendering a list of items please

File: src/components/comments-container.jsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+        {currentArticleComments.map((comment) => {
+          return (
+        <div>
+        <CommentContainer comment={comment}/>
```
Comment (P-Copley) [inline src/components/comments-container.jsx]: Another `div` that can be taken out 😁

**Comment by P-Copley** [comment]
Hey @BobbyWhaler 👋 

There's a few things to look at on this App:
- What's the purpose of the `AppBody` component? Seems like you're not using `App` for anything and have made an extra component to do it's job? There's no plan in this repo I can see so unless you've got this for later use that I'm not aware of remove it and just use `App`.
- Everything is on a single page here, is this intentional? When clicking on an article it doesn't show me any more information than it did before. If this is the case why bother clicking on it? You could do something like removing the article bodies from the list of all articles so you can pick one based on title and then click on it to read it but there should be a point to clicking on it or not bother.

The implementation of viewing comments works with this App design but it seems like an odd choice to view a list of comments on the right whilst being able to scroll away from that article. If you're happy with this design then go ahead and merge otherwise submit a new PR if you want us to have a look at the refactor.

**Review by MRombi — APPROVED** [review]
Looks good to me, good to merge 😄

**Pull request merged**